install/kubernetes/cilium/charts/hubble-tls/templates/_helpers.tpl

```diff
@@ -0,0 +1,29 @@
+{{/*
+Generate TLS certificates for Hubble server and Hubble Relay.
+*/}}
+{{- define "ca.gen-certs" }}
+{{- $ca := .ca | default (genCA "hubble-ca.cilium.io" 1095) -}}
+{{- $_ := set . "ca" $ca -}}
```

Comment:
Similar as above. `.ca` does not seem used (neither for writing or for reading), why do we have to set it?

Reply:
See reply above.